### ISSUE TITLE
Tables with borders don't get correct width and height in chrome

### ISSUE THREAD

**Issue by Roystbeef** [tsayen/dom-to-image#167]
### Use case: description, code  

If I have a table's width/height set to 100% and a border applied to it, the table in the outputted image is short. This is because chrome implicitly sets all tables to be `box-sizing: border-box` but the width/height obtained from `window.getComputedStyle()` assumes the table is set to `box-sizing: content-box`. This works in firefox bc firefox supports `content-box` for tables.

[jsfiddle](https://jsfiddle.net/ercg0kj2/)

### Expected behavior
tables in the image should match tables in the dom

### Actual behavior (stack traces, console logs etc)
the table computes the wrong width/height

### Browsers

- [x] Chrome 49+  
- [ ] Firefox 45+

**Comment by hakimio** [comment]
@Roystbeef Did you ever find a solution for this?

**Comment by Roystbeef** [comment]
nope. from the fiddle it looks like this is still a problem 😢. Explicitly setting the table to `box-sizing: border-box` however, does bring the domToImage result in line with what is rendered.
